AdyenCard/Form/FormCardNumberItem.swift

```diff
@@ -74,52 +86,58 @@ internal final class FormCardNumberItem: FormTextItem, Observer {
     }
     
     /// Updates the item with the detected brands.
-    /// and sets the first supported one as the `currentBrand`.
+    /// and sets the first supported one as the `initialBrand`.
     internal func update(brands: [CardBrand]) {
         detectedBrands = brands
+        
         switch (brands.count, brands.first(where: \.isSupported)) {
         case (1, _):
-            update(currentBrand: brands.first)
+            update(initialBrand: brands.first)
         case let (2, .some(firstSupportedBrand)):
-            update(currentBrand: firstSupportedBrand)
+            update(initialBrand: firstSupportedBrand)
         case (2, nil):
             // if there are 2 brands and neither is supported,
             // need to show unsupported text
-            update(currentBrand: nil, defaultSupportedValue: false)
+            update(initialBrand: nil, defaultSupportedValue: false)
         default:
-            update(currentBrand: nil)
+            update(initialBrand: nil)
         }
         
+        isDualBranded = brands.count == 2 && brands.allSatisfy(\.isSupported)
```

Comment:
maybe `>= 2`? its highly unlikely that its more than 2 brands, but are we sure its impossible?

Reply:
We are told so. And if by some bug there are 3, may not be right to call it dual-branded either :)